dependencies.yaml

```
@@ -60,7 +60,7 @@ dependencies:
       - output_types: [conda, pyproject, requirements]
         packages:
           - cmake>=3.24.0,!=3.30.0
-          - &legate_core legate-core==24.06.*,>=24.06.01
+          - &legate legate==24.09.00.dev290
```

```suggestion
          - &legate legate=24.09.*,>=0.0.0dev0
```

Instead of `==` pinning to a specific nightly, I think it'd be better to depend let this float within all `24.09.*` packages, so every run of CI here is building against the latest `legate` nightly. Would you consider that?

---

And I think the `experimental` should be added to this list:

https://github.com/rapidsai/legate-boost/blob/fea797e58718edb27e9ab5a054edeaa476ac047d/dependencies.yaml#L48-L51

Like this:

```yaml
 channels: 
   - legate 
   - legate/label/experimental
   - conda-forge 
   - nvidia
```

That way, nightlies will be correctly found in any places using `rapids-dependency-file-generator` to create a conda environment, like the build instructions:

https://github.com/rapidsai/legate-boost/blob/fea797e58718edb27e9ab5a054edeaa476ac047d/contributing.md#L8-L11

---

OK, I think that makes sense.  I wasn't sure what to pick here and was briefly considering it (but also wouldn't have know the `>=0.0.0dev0` trick ;)).

I won't be surprised if this breaks every week on something.  But maybe we best just roll with it and naturally settle into the stable release with what you suggested.

---

Lets try James' solution. If it becomes too much trouble we can pin.

---

Yes, I fully agree we need to test with the nightlies.  The "known good version" can just be a tag when we need it, I guess.